src/Common/src/TypeSystem/IL/HelperExtensions.cs

```diff
@@ -20,9 +20,16 @@ public static MetadataType GetHelperType(this TypeSystemContext context, string
 
         public static MethodDesc GetHelperEntryPoint(this TypeSystemContext context, string typeName, string methodName)
         {
-            MetadataType helperType = context.GetHelperType(typeName);
-            MethodDesc helperMethod = helperType.GetKnownMethod(methodName, null);
-            return helperMethod;
+            try
+            {
+                MetadataType helperType = context.GetHelperType(typeName);
+                MethodDesc helperMethod = helperType.GetKnownMethod(methodName, null);
+                return helperMethod;
+            }
+            catch (InvalidOperationException ex)
+            {
+                throw new InvalidOperationException($"Cannot find type {typeName} to obtain compiler helper method {methodName}", ex);
```

Comment:
This error message is not right for the case where `GetKnownMethod` throws `InvalidOperationException` above.

Reply:
I update the message. Now it should cover both cases.